runtime/src/accounts_db.rs

```diff
@@ -356,6 +356,21 @@ impl<'a> Serialize for AccountsDBSerialize<'a> {
     }
 }
 
+#[derive(Clone, Default, Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
Actually I do use it because i just package these things in the snapshot to avoid touching that code. They aren't strictly needed but an extra 30-ish bytes probably isn't going to affect it.